src/main/scala/zio/zmx/diagnostics/ZMXServer.scala

```diff
@@ -110,7 +106,7 @@ private[zmx] object ZMXServer {
       _ <- client.close
     } yield message
 
-  def make(config: ZMXConfig): ZIO[Clock with Console with ZMXParser with FiberDumpProvider, Exception, ZMXServer] = {
+  def make(config: ZMXConfig): ZManaged[Clock with Console with ZMXParser with FiberDumpProvider, Exception, Unit] = {
```

Comment:
not sure about the Unit @adamgfraser

Reply:
I think `Unit` makes sense here. The `acquire` action of this `ZManaged` just performs the effect of starting this concurrent process and then the `release` action shuts it down so using `ZManaged` here is valuable to control the scope even thought doesn't actually return anything.